src/devices/Display/samples/Large4Digit7SegmentDisplay.sample.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.Display;
+
+namespace SevenSegmentSample
+{
+    class Program
+    {
+        const string SupportedCharacters = "0123456789aAbBcCdDeEfFgGhHiIjJlLnNoOpPrRsStuUyYzZ-=_|°[]     ";
+
+        static void Main(string[] args)
+        {
+            // Initialize display (busId = 1 for Raspberry Pi 2 & 3)
+            using var display = new Large4Digit7SegmentDisplay(I2cDevice.Create(new I2cConnectionSettings(busId: 1, Ht16k33.DefaultI2cAddress)))
+            {
+                // Set max brightness
+                Brightness = Ht16k33.MaxBrightness
+            };
+
+            // Write "Iot" on the display
+            display.Write("Iot");
+
+            // Wait 2 seconds
+            Thread.Sleep(2000);
+
+            // Change 'o' to '°' in "IoT"
+            display[1] = (Segment)FontHelper.GetCharacter('°');
+
+            // Flush display buffer (needs to be done explicitly when not calling Write or Dot)
+            display.Flush();
```

Comment:
Considering you enable buffering later this shouldn't be needed, right?

Reply:
I deliberately left the this operator as is not to flush the buffer automatically.